google_redis/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 3.0"
+    }
+  }
+  required_version = ">= 1.0.11"
```

Comment:
For Terraform versions is it better to use `>=` or `~>`?

Reply:
Good point, changed to `~>`